### ISSUE TITLE
[Accessibility][Name, Role, Value - Property Pane_Ask a question (Bot Asks)]: Name is not defined for up and down arrow in 'Max Turn count' spin button.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2117]
**User Experience** 
Users who rely on AT will might get confused if the controls name is not defined for the available controls.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and turn on narrator.
2. Navigate to already created dialog tree item and activate it. Such as- Greetings, Actions, EndDialog etc.
3. Click on "+" and select 'Ask a question'. Then select any of the available options except Oauth login.
4. Navigate to property pane appeared at right side of the screen.
5. Now, navigate to 'Max turn count' spin button.
6. Observe the issue.

**Actual:** 
The name is not defined for the up/down arrow in the 'Max Turn count' spin button.
Narrator only narrates their role as button on getting the focus in scan mode.

**Expected:**
The name should be defined for the controls.
Narrator should announce increase and decrease button.

**Note:**
Same issue observed at every property pane which have spin button such as "Connect to QnA Knowledgebase".

**MAS Impacted:** [MAS4.1.2](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Attachment:**
![MAS4 1 2_Name is not defined for up and down key](https://user-images.githubusercontent.com/60913131/75454741-e9784e00-599b-11ea-9b93-477648aa5ad4.png)

**Comment by beyackle** [comment]
I'm not sure what to do about this; that control no longer exists, and the field that's there now is properly labeled. Do we have a policy on these?

**Comment by beyackle** [comment]
The up/down arrow control no longer exists, so this is now a non-issue.

**Comment by ashish315** [comment]
@beyackle Still we have up/down key. We have checked in 'Master' version of Bot Composer.

![image](https://user-images.githubusercontent.com/60913131/78450416-0e976500-769c-11ea-8c82-18fb94681c6e.png)

**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
Browser: Microsoft Edge Canary {Version 83.0.474.0 (Official build) canary (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Comment by beyackle** [comment]
We have the arrows back after all, and SpinButton supports aria labels for increment and decrement buttons, so I'm adding those in as part of another PR to put aria labels on things.

**Comment by ashish315** [comment]
As checked and verified on below environment, the issue is fixed now. So, we are closing the issue.

**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
